packages/neuron-wallet/src/controllers/sync-api.ts

```diff
@@ -224,6 +227,14 @@ export default class SyncApiController {
     SyncApiController.emiter.on('cache-tip-block-updated', async states => {
       const newSyncState = await this.#estimate(states)
       this.#syncedBlockNumber.setNextBlock(BigInt(newSyncState.cacheTipNumber))
+      if (
+        env.app.isPackaged &&
+        process.env.CKB_NODE_ASSUME_VALID_TARGET &&
+        (newSyncState.cacheTipNumber === 0 || newSyncState.cacheTipNumber === this.#lastCacheTipNumber)
```

Comment:
Would `newSyncState.cacheTipNumber === this.#lastCacheTipNumber` be true when synchronization is done?

Reply:
That's right, it shouldn't equal bestKnownBlockNumber

Reply:
If synchronization is done, it wouldn't subscribe `cache-tip-block-updated` event?

Reply:
> If synchronization is done, it wouldn't subscribe `cache-tip-block-updated` event?

It keeps subscribing to `cache-tip-block-updated` since `synchronization is done` is temporary.

Reply:
I think I should subscribe ckb logs to setting status. I will reimplement it

Reply:
I fixed it with validate `bestKnownBlockTimestamp` is within 24 hours